light-zk.js/src/wallet/provider.ts

```diff
@@ -172,24 +173,64 @@ export class Provider {
 
   private async fetchLookupTable() {
     try {
-      if (!this.wallet.isNodeWallet) {
-        const response = await axios.get("http://localhost:3331/lookuptable");
-        const lookUpTable = new PublicKey(response.data.data);
-        this.lookUpTable = lookUpTable;
-        this.relayer.accounts.lookUpTable = lookUpTable;
-        return;
+      if (this.wallet.isNodeWallet) {
+        await this.fetchLookupTableNodeWallet();
+      } else {
+        await this.fetchLookupTableExternal();
       }
-      if (!this.provider) throw new Error("No provider set.");
-      // TODO: remove this should not exist
-      const lookUpTable = await initLookUpTable(this.wallet, this.provider);
-      this.lookUpTable = lookUpTable;
-      this.relayer.accounts.lookUpTable = lookUpTable;
     } catch (err) {
       console.error(err);
       throw err;
     }
   }
 
+  private async fetchLookupTableExternal() {
+    const response = await axios.get("http://localhost:3331/lookuptable");
```

Comment:
I don't think we should hard code this address? Can we make it configurable? I guess that either a fn parameter or an environment variable would make sense.

Reply:
agreed

Reply:
changed to this.url + "/lookuptable"